src/service/document-info-impl.js

```diff
@@ -88,6 +92,7 @@ export class DocInfo {
         : sourceUrl;
     }
     const pageViewId = getPageViewId(ampdoc.win);
+    const pageViewId64 = getRandomString64(ampdoc.win);
```

Comment:
This line is racy since this code path eventually calls `extensionsFor()` in `crypto-impl.js`: https://github.com/ampproject/amphtml/blob/fb19bc7f03eaac3574183fede4f23ebc03b3b7ae/src/service/crypto-impl.js#L151-L158

The "extensions" service, however, may not be registered on the window yet. I think this is the cause of new runtime error #23988.

Reply:
Thanks for reporting. Is there a way to wait for the extensions service to be ready? 
In the meantime, shall we revert this change?

Reply:
#23998 should fix this.